projectmanagement/src/test/groovy/life/qbic/projectmanagement/domain/project/experiment/VariableLevelSpec.groovy

```diff
@@ -12,18 +12,18 @@ import spock.lang.Specification
  */
 class VariableLevelSpec extends Specification {
 
-    def "When at least one variable value is not part of a Variable level, throw an exception"() {
-        given:
-        ExperimentalValue value = ExperimentalValue.create("test", "unit")
-        ExperimentalValue unknownValue = ExperimentalValue.create("unknown", "nothing")
-        ExperimentalVariable variable = ExperimentalVariable.create("testVariable", value)
-
-        when:
-        new VariableLevel(variable, unknownValue)
-
-        then:
-        thrown(UnknownVariableLevelException)
-    }
+//    def "When at least one variable value is not part of a Variable level, throw an exception"() {
+//        given:
+//        ExperimentalValue value = ExperimentalValue.create("test", "unit")
+//        ExperimentalValue unknownValue = ExperimentalValue.create("unknown", "nothing")
+//        ExperimentalVariable variable = ExperimentalVariable.create("testVariable", value)
+//
+//        when:
+//        new VariableLevel(variable, unknownValue)
+//
+//        then:
+//        thrown(UnknownVariableLevelException)
+//    }
```

Comment:
why is that commented out?

Reply:
ah because this is nothing the level does, it is something the variable does

Reply:
the level no longer gets to know the whole variable but only the name. when creating a level the aggregate is in charge to make sure the variable is defined in the experimental design and can have the requested level